qemu/tests/throttle_operation_test.py

```diff
@@ -82,6 +83,14 @@ def operation_hotplug():
         plug.hotplug_devs_serial("stg3")
         plug.hotplug_devs_serial("stg4")
 
+    def operation_snapshot():
```

Comment:
suggest to update test steps(line 29) with the newly added snapshot+throttle testing